src/views/pages/accounts/LoginPageTs.ts

```diff
@@ -233,9 +233,10 @@ export default class LoginPageTs extends Vue {
     if (accountPass !== passwordHash) {
       return this.$store.dispatch('notification/ADD_ERROR', NotificationType.WRONG_PASSWORD_ERROR)
     }
-
+    // the profile which is created by import private key has no seed but wallet
+    const shouldPerfectProfile: boolean = (!account.values.has('seed') || !account.values.get('seed').length) && knownWallets.size === 0
     // if account setup was not finalized, redirect
-    if (!account.values.has('seed') || !account.values.get('seed').length) {
+    if (shouldPerfectProfile) {
```

Comment:
what does `perfect profile` mean?

Reply:
if we want create a profile, we will go by several steps. 
for example, there are 3 steps to create a profile.
i finished the 1st step what create the profileA and i exit our web app.
now, the profileA is the `shouldPerfectProfile`.